var/spack/repos/builtin/packages/simgrid/package.py

```diff
@@ -145,7 +146,8 @@ def cmake_args(self):
             args.append("-Denable_documentation=OFF")
         if spec.satisfies("+mc"):
             args.append("-Denable_model-checking=ON")
-
+        if spec.satisfies("+msg"):
```

Comment:
I realize the pattern is already established in this package, but did you know there are built-in methods that can help set `cmake` arguments from variants?  Check out https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake.

Reply:
You can see more on `define` and `define_from_variant` starting at https://spack.readthedocs.io/en/latest/spack.build_systems.html?highlight=cmakepackage#spack.build_systems.cmake.CMakePackage.define.